README.md

```diff
@@ -37,13 +37,15 @@ The easiest way to get started using ct is to download one of the binaries from
 To build from source you'll need to have the go compiler installed on your system.
 
 ```shell
-git clone --branch v0.1.0 https://github.com/coreos/container-linux-config-transpiler
+git clone --branch v0.3.1 https://github.com/coreos/container-linux-config-transpiler
 cd container-linux-config-transpiler
 ./build
```

Comment:
While, you're at it, I think we added `make build` / `make release` so we might want to say that here. The `build` script had to be kept because something was written depending on it I think. cc @dgonyeo